package.json

```diff
@@ -160,5 +160,8 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
+  },
+  "dependencies": {
```

Comment:
Please revert this change in package.json and also the yarn.lock changes.

Reply:
Done.
However, I have failed the CI/ node latest with browserstack (pull request) test. I have a "Error: Configuration parameter username is required." in the log for that test. I don't know what I should do (provide a user name, how??) in this case.

Reply:
The failing browserstack test is normal. This happens because forks can not use the secrets in our repo due to security reasons and so the test can not read the credentials from the repo setting on our side.

After the changes are merged a new CI build is done in the repo which runs the browserstack test.

So nothing to worry about.